src/create-config.ts

```diff
@@ -0,0 +1,317 @@
+
+const questions = (choices: Choice[]): PromptObject[] => {
+  return [
+    {
+      type: 'autocomplete',
+      name: 'apiName',
+      message: 'What is the name of the API Integration?',
+      choices: choices,
+    },
+    {
+      type: 'confirm',
+      name: 'airnodeAuthorizer',
+      message: 'Do you want to include the Airnode Authorizer?',
+      initial: false,
+    },
+    {
+      type: 'confirm',
+      name: 'airnodeHeartbeat',
+      message: 'Do you want to enable the Airnode Heartbeat?',
+      initial: false,
+    },
+  ];
+};
+
+const main = async () => {
+  const operationsRepository = readOperationsRepository();
+  const apiChoices = Object.keys(operationsRepository.apis).map((api) => ({ title: api, value: api })) as Choice[];
+  const response = await promptQuestions(questions(apiChoices));
+  const apiData = operationsRepository.apis[response.apiName];
+
+  //// Build Config.json  ////
+
+  // Get all the chains the API will be deployed on
+  const apiChains = [
+    ...new Set(
+      Object.keys(apiData.beacons).flatMap((beacon) => apiData.beacons[beacon].chains.map((chain) => chain.name))
+    ),
+  ];
+
+  const chains = apiChains.map((chainName) => {
+    const chainId = chainNameToChainId[chainName];
+    //TODO: Add RequesterAuthorizerWithManager
+    const authorizers =
+      response.airnodeAuthorizer && RequesterAuthorizerWithAirnodeAddresses[chainId]
+        ? [RequesterAuthorizerWithAirnodeAddresses[chainId]]
+        : [];
+    const contracts = { AirnodeRrp: AirnodeRrpAddresses[chainId] || '' };
+    const options = {
+      txType: 'eip1559' as const,
+      priorityFee: {
+        value: 3.12,
+        unit: 'gwei' as const,
+      },
+      baseFeeMultiplier: 2,
+    };
+    const providers = {
+      provider1: {
+        url: `\${${chainName}_PROVIDER_URL}`.toUpperCase(),
+      },
+    };
+
+    return {
+      authorizers,
+      contracts,
+      id: `${chainId}`,
+      providers,
+      type: 'evm' as const,
+      options,
+      maxConcurrency: 100,
+      blockHistoryLimit: 300,
+      minConfirmations: 0,
+    };
+  });
+
+  const nodeSettings = {
+    nodeVersion: require('@api3/airnode-node/package.json').version,
+    cloudProvider: {
+      type: 'aws' as const,
+      region: 'us-east-1' as const,
+      disableConcurrencyReservations: true,
+    },
+    airnodeWalletMnemonic: '${AIRNODE_WALLET_MNEMONIC}',
+    heartbeat: {
+      enabled: response.airnodeHeartbeat,
+      ...(response.airnodeHeartbeat && {
+        apiKey: '${HEARTBEAT_API_KEY}',
+        id: '${HEARTBEAT_ID}',
+        url: '${HEARTBEAT_URL}',
+      }),
+    },
+    httpGateway: {
+      enabled: true,
+      apiKey: '${HTTP_GATEWAY_API_KEY}',
+      maxConcurrency: 20,
+    },
+    httpSignedDataGateway: {
+      enabled: true,
+      apiKey: '${HTTP_SIGNED_DATA_GATEWAY_API_KEY}',
+      maxConcurrency: 20,
+    },
+    logFormat: 'plain' as const,
+    logLevel: 'INFO' as const,
+    stage: 'dev' as const,
```

Comment:
This isn't a typed union, so you don't need const for it.
`logFormat` and `logLevel` are correct.